src/Calendar.js

```diff
@@ -70,6 +71,11 @@ const Calendar = ({
     return `${month} ${year}`;
   };
 
+  const isBeforeThisMonth = isNewMonth => {
+    const date = getDate(!isNewMonth);
```

Comment:
I used this `getDate()` function but I have no idea how this is working only by giving odd or even month. disabling the previous button is still buggy due to this. any idea how to fix it?

Reply:
IDK why you're struggling with this step. It's rather simple and there's no need for such functions. I added this and it works fine:
![image](https://user-images.githubusercontent.com/20098648/59438792-6614cd80-8e09-11e9-88c6-a82ce55cc65e.png)
![image](https://user-images.githubusercontent.com/20098648/59438818-70cf6280-8e09-11e9-9d23-51bf475d9651.png)
The only problem that it has is because of how `isBeforeDate` works, it disables today as well. For now, you can leave it like this(by checking disable backward prop of course). I can change how isBeforeDate works ultimately

Reply:
There's no need to disable the previous arrow for now. Maybe I can add it later

Reply:
But we need it for our project, there is no easy way you say?

Reply:
Yeah, you're right. It's necessary. To do this you need to toggle a `-disabled` class for the previous month arrow(as you've added perfectly now), But where and how you should do this? the answer is in `handleAnimationEnd` function(It runs after month changes). for applying this change you can compare `today`'s month with `activeDate`'s month. and If they're the same you can toggle the `-disabled` class.